scipy/interpolate/interpolate.py

```diff
@@ -1464,6 +1464,33 @@ class RegularGridInterpolator(object):
 
     .. versionadded:: 0.14
 
+    Example
```

Comment:
This should be ``Examples``. That's the section header name recognized by the doc builder. (and add the corresponding underscore on the next line)